src/components/profile/profile.js

```diff
@@ -187,10 +190,23 @@ export class Profile extends React.Component {
                 {this.state.data.handle ? "@" : null}
                 {this.state.data.handle}
               </Card.Meta>
-              <Card.Description textAlign="center"> {data.description}</Card.Description>
+              <Card.Description style={{ marginTop: "1em" }}>
+                {data.branch}
+              </Card.Description>
+              <Card.Description style={{ marginBottom: "1em" }}>
+                {data.degreeSem}
+              </Card.Description>
+              <p styleName="style.gray">{data.description}</p>
             </div>
-            <div className="center aligned content" style={{ border: "0", textAlign: "center" }}>
-              <LinkDisplay handle={this.props.handle} theme={this.props.theme} />
+            <div
+              className="center aligned content"
+              style={{ border: "0", textAlign: "center" }}
+            >
+              <LinkDisplay
```

Comment:
You rendered the email above, right? What is this for? I am sorry I don't understand this.

Reply:
Above? No, email is to be rendered along with other social links. For this purpose, I am fetching email from the `Profile` model, in the `Profile` component and passing it as a prop to the `LinkDisplay` component

Reply:
What is [this](https://github.com/IMGIITRoorkee/omniport-frontend-student-profile/pull/21/files#diff-724fa2c13a556bdd1be55a194eb7b73b7df6c5aa2395cfea57f3b4c938bf0ad3R96) for?

Reply:
![Screenshot_2020-11-22 imgian1](https://user-images.githubusercontent.com/25002246/99910801-15034b80-2d16-11eb-943b-f0358533cd66.png)

For rendering this email icon (first) along with other social links.